BraveShared/Extensions/SharedExtensions.swift

```diff
@@ -16,3 +16,9 @@ extension Date {
         return UInt64(self.timeIntervalSince1970 * 1000)
     }
 }
+
+extension Timestamp {
+    public func toDate() -> Date {
```

Comment:
Add unit test for it

Reply:
Certainly need a test here, this will guarantee us, we are displaying proper created time in password details

Reply:
Test adde at https://github.com/brave/brave-ios/pull/5013/commits/62fb5d07c7cb75c3e04c45a580d1ac7d915a8825